### PR TITLE
(DOCS-6148) Clarify app registration

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do? What is the motivation?
- Clarify that app registration must take place first
- Provide links for app registration

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/integrations/guide/azure-programmatic-management.md [hunk 1]
```diff
@@ -12,17 +12,15 @@ further_reading:
...
+The standard Azure integration uses an app registration credential process for implementing metric collection, and an Azure Event Hub setup for sending Azure Platform Logs. Create the app registration in Azure before integrating Datadog with your Azure environment, and configure it with the **Monitoring Reader** permission for Datadog to monitor the provided scope (subscriptions or management groups). If you don't already have an app registration created, see [Integrating through the Azure Portal][6] or [Integrating through the Azure CLI][4] for setup instructions.
 
-If your Datadog account is hosted on any other [Datadog site][3], use the standard [Azure integration][5] to collect monitoring data from your Azure environment.
-
-If you prefer to use CLI, see the [Azure CLI for Datadog][2].
+**Note**: You can assign read permissions at the management group level when creating the app registration in Azure, to monitor multiple subscriptions and have new subscriptions in the management group automatically monitored.
```
Comment (alai97) [inline content/en/integrations/guide/azure-programmatic-management.md:23]:
```suggestion
You can assign read permissions at the management group level when creating the app registration in Azure, to monitor multiple subscriptions and have new subscriptions in the management group automatically monitored.
```

File: content/en/integrations/guide/azure-programmatic-management.md [hunk 2]
```diff
@@ -12,17 +12,15 @@ further_reading:
...
 
-**US3**: If your organization is on the Datadog US3 site, you can use the Azure Native integration to streamline management and data collection for your Azure environment. Datadog recommends using this method when possible. Setup entails creating a [Datadog resource in Azure][14] to link your Azure subscriptions to your Datadog organization. This replaces the app registration credential process for metric collection and Event Hub setup for log forwarding.
+**US3**: If your organization is on the Datadog US3 site, you can use the Azure Native integration to streamline management and data collection for your Azure environment. Datadog recommends using this method when possible. Setup entails creating a [Datadog resource in Azure][14] to link your Azure subscriptions to your Datadog organization. This replaces the app registration credential process for metric collection and Event Hub setup for log forwarding. See the [Managing the Azure Native Integration][1] guide for more information.
```
Comment (alai97) [inline content/en/integrations/guide/azure-programmatic-management.md:17]: Just curious here, is there a reason we are not using the[ site region shortcode](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1183023503/Regions+Sites#Links-in-the-Site-Region-Shortcode) to nest this information when a user clicks on US3 in the Docs Site?

```suggestion
**US3**: If your organization is on the Datadog US3 site, you can use the Azure Native integration to streamline management and data collection for your Azure environment. Datadog recommends using this method when possible. Setup entails creating a [Datadog resource in Azure][14] to link your Azure subscriptions to your Datadog organization. This replaces the app registration credential process for metric collection and Event Hub setup for log forwarding. See the [Managing the Azure Native Integration guide][1] for more information.
```

File: content/en/integrations/guide/azure-programmatic-management.md [hunk 3]
```diff
@@ -133,21 +131,21 @@ See the [Virtual Machine Extension resource][10] in the Terraform registry for m
...
 [12]: /agent/guide/agent-configuration-files/?tab=agentv6v7
 [13]: https://www.terraform.io
 [14]: https://learn.microsoft.com/en-us/azure/partner-solutions/datadog/overview
 [15]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs
-[16]: https://learn.microsoft.com/en-us/cli/azure/monitor/diagnostic-settings?view=azure-cli-latest#az-monitor-diagnostic-settings-create
 [17]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/integration_azure
 [18]: /logs/guide/azure-logging-guide
 [19]: https://app.datadoghq.com/monitors/recommended
 [20]: /monitors/notify/#notify-your-team
+[21]: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/service_principal_client_secret#creating-a-service-principal
```
Comment (alai97) [inline content/en/integrations/guide/azure-programmatic-management.md:151]: Do you intend on adding this link to a section above?

**Review by alai97 — COMMENTED** [review]
Some suggestions!

File: content/en/integrations/guide/azure-programmatic-management.md (hunk 2 above)
Comment (buraizu) [reply content/en/integrations/guide/azure-programmatic-management.md:17]: This was part of the recent Azure doc reorganization that moved away from the site short codes approach.

File: content/en/integrations/guide/azure-programmatic-management.md (hunk 1 above)
Comment (buraizu) [reply content/en/integrations/guide/azure-programmatic-management.md:23]: The note is intended to emphasize the point, is there a particular reason to suggest its removal?

File: content/en/integrations/guide/azure-programmatic-management.md (hunk 3 above)
Comment (buraizu) [reply content/en/integrations/guide/azure-programmatic-management.md]: Thanks for flagging, removed!

File: content/en/integrations/guide/azure-programmatic-management.md (hunk 1 above)
Comment (alai97) [reply content/en/integrations/guide/azure-programmatic-management.md:23]: I find that with "You can..." sentences, you don't need the **Note:** as it is the start of a new sentence, feel free to disregard this suggestion!

**Review by alai97 — APPROVED** [review]

File: content/en/integrations/guide/azure-programmatic-management.md (hunk 1 above)
Comment (buraizu) [reply content/en/integrations/guide/azure-programmatic-management.md:23]: Thanks for clarifying, I'll keep the note for now to retain the emphasis, but I'll keep that feedback in mind going forward :bow:

**Pull request merged**